### ISSUE TITLE
Would it be possible to add a bit more contrast for the currently visible tab in the Light variant ?

### ISSUE THREAD

**Issue by max-g** [Railly/one-hunter-vscode#12]
Really nice theme, I was wondering if you could make the current opened tab a bit more visible in the Light variant ?

It's kind of hard to see which one is currently visible. In the following screenshot SiteFooter.tsx is the one currently visible :

![Screen Shot 2023-01-10 at 11 53 34](https://user-images.githubusercontent.com/2347985/211613480-2204019c-5df0-480e-93f7-4ecdb8afbcb6.png)

**Comment by Railly** [comment]
Sorry, I didn't see the notification of this issue. Sure, I'll look into it. If there were any other suggestions, feel free to share them with me.

**Comment by max-g** [comment]
It's totally subjective, but i like having the accent color of the theme as the top or lower border of the active tab, like in the Github and Ayu themes. I was thinking maybe the pink accent color would be nice.

eg: 
<img width="352" alt="Screenshot 2023-01-29 at 20 51 04" src="https://user-images.githubusercontent.com/2347985/215371787-ce0bbc82-a731-440f-8698-88264280b442.png">
<img width="285" alt="Screenshot 2023-01-29 at 20 50 28" src="https://user-images.githubusercontent.com/2347985/215371799-f0334ca5-0370-4113-a654-4e4919ea3530.png">

**Comment by Railly** [comment]
Yup, I fully agree with you. I'll check out other light themes as well, because tbh I made this one really fast. 
Annotated! 📝

**Comment by max-g** [comment]
I just realized that there is a setting that will output a top border for unsaved file :   
![Screenshot 2023-01-31 at 10 48 10](https://user-images.githubusercontent.com/2347985/215808894-3a4edfcc-7003-4163-962a-80581ef95060.png)

Here is how it shows on Github Light theme :
- Current active tab == tab with unsaved changes :
![Screenshot 2023-01-31 at 10 50 51](https://user-images.githubusercontent.com/2347985/215809605-b7e5ee35-80f8-4938-9fdd-3ae369417759.png)

- Current active tab != tab with unsaved changes : 
![Screenshot 2023-01-31 at 10 49 58](https://user-images.githubusercontent.com/2347985/215809656-aff57dc3-ca8c-47a0-a696-e768737c3d47.png)

Here is how it shows on Ayu Light theme :
- Current active tab == tab with unsaved changes :
![Screenshot 2023-01-31 at 10 50 44](https://user-images.githubusercontent.com/2347985/215809778-1a258ceb-bd71-4977-ab86-3735a1ea163f.png)
- Current active tab != tab with unsaved changes : 
![Screenshot 2023-01-31 at 10 50 08](https://user-images.githubusercontent.com/2347985/215809818-74c93a16-7b7b-43df-a707-cdb8c3138813.png)
